docs/en/siem/rules-ui-create.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[rules-ui-create]]
+[role="xpack"]
+=== Managing detection engine rules
+
+On the Rules page, you can create, clone, delete, enable, import, and export 
+rules. Rules run periodically, and search the specified indices for 
+documents that meet their criteria.
+
+[role="screenshot"]
+image::all-rules.png[]
+
+[float]
+==== Create a new rule
+
+. Go to *SIEM* -> *Detection Engine* -> *Manage rules*.
+. Click *Create new rule*.
++
+The *Create new rule* page is displayed.
+[role="screenshot"]
+image::create-new-rule.png[]
+. Set which indices the rule analyzes for signals:
++
+.. Configured SIEM indices (*Management* -> *Advanced 
+setting* -> *siem:defaultIndex*).
+.. A custom list of indices.
+. Use the filter and query fields to create the criteria used to produce 
+signals. For example, a rule that detects when a Microsoft Office product 
+starts a process in a user folder uses the following query and filters:
+*Ben: add example*
+. Click *Continue*.
++
+The *About rule* pane is displayed.
+[role="screenshot"]
+image::about-rule-pane.png[]
+. Fill in the following fields:
+.. _Name_: The rule's name.
+.. _Description_: A description of what the rule does.
+.. _Severity_: Select the severity levels of signals created by the rule:
+... `Low`: Signals that are of interest but generally not considered to be 
+security incidents. Sometimes, a combination of low severity events can 
+indicate suspicious activity.
+... `Medium`: Signals that require investigation.
+... `High`: Signals that require an immediate investigation.
+... `Critical`: Signals that indicate a security incident has occurred.
+.. _Risk score_: A numerical value between 0 and 100 that correlates with the _Severity_ level. General guidelines are:
+... `21` represents low severity.
+... `47` represents medium severity.
+... `73` represents high severity.
+... `99` represents critical severity.
+.. _Reference URLs_ (optional): References to information that is relevant to 
+the rule. For example, links to the relevant Mitre URL.
+.. _False positives_ (optional): List of common scenarios that may produce 
+false-positive signals.
+.. _Tags_ (optional): Words and phrases that can be used to categorize, filter, 
```

Comment:
Change from passive to active, "Words and phrases used to categorize, filter, and search the rule".